dev/breeze/tests/test_pytest_args_for_test_types.py

```diff
@@ -19,9 +19,18 @@
 import pytest
 
 from airflow_breeze.global_constants import GroupOfTests
+from airflow_breeze.utils.path_utils import AIRFLOW_SOURCES_ROOT
 from airflow_breeze.utils.run_tests import convert_parallel_types_to_folders, convert_test_type_to_pytest_args
 
 
+def _all_new_providers() -> list[str]:
```

Comment:
The term "new" is always legacy as soon as the PR is merged. Can you have a neutral name or also mark it with TODO that at point of removal of "old" provider structure the function is renamed? (Because  understand that today the name `_all_providers` would be mis-leading... or how about `_all_self_contained_providers()`?

Reply:
I want to change it afterwards - I deliberately leave "new" now when we have both "new" and "old" but - when I remove the "old" (which is pretty soon) the "new"will become "the only" and I will remove "new" then.

Reply:
But yeah. I will add todos :)